TheMovieDb/New Group/Controllers/SearchResultViewControllerç.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SearchResultViewController.swift
+//  TheMovieDb
+//
+//  Created by Alex on 23/11/21.
+//
+
+import UIKit
+
+final class SearchResultViewController: UIViewController {
+ 
+    var isFiltering: Bool = false
+    var filteredMoviesCount: Int = 0
+    var moviesCount: Int = 0
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    
+}
+
+extension SearchResultViewController: UITableViewDelegate, UITableViewDataSource {
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        return UITableViewCell()
+    }
+    
+    func tableView(_ tableView: UITableView,
+                   numberOfRowsInSection section: Int) -> Int {
+      if isFiltering {
+        return filteredMoviesCount
```

Comment:
Nitpick: Indentation

Reply:
I've removed this file, because it got created by error when I was managing the project folder and it made a duplicate of the original SearchResultViewController.